# ALL:ALL - Se fuerza la compilacion y exportacion estatica de vulkan-loader en CMakeLists.txt para que funcione el enlace estatico de ffmpeg.
--- vulkan-loader/loader/CMakeLists.txt
+++ vulkan-loader/loader/CMakeLists.txt
@@ -415,7 +415,7 @@
         message(WARNING "The APPLE_STATIC_LOADER option has been set. Note that this will only work on MacOS and is not supported "
                 "or tested as part of the loader. Use it at your own risk.")
     else()
-        add_library(vulkan SHARED)
+        add_library(vulkan STATIC)
     endif()
 
     target_sources(vulkan PRIVATE ${NORMAL_LOADER_SRCS})
@@ -529,9 +529,8 @@
 endif()
 
 # Generate CMake Configuration File (IE: VulkanLoaderConfig.cmake)
-install(TARGETS vulkan EXPORT VulkanLoaderConfig)
+install(TARGETS vulkan DESTINATION lib)
 set_target_properties(vulkan PROPERTIES EXPORT_NAME "Loader")
-install(EXPORT VulkanLoaderConfig DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/VulkanLoader NAMESPACE Vulkan::)
 
 # Generate CMake Version File (IE: VulkanLoaderConfigVersion.cmake)
 include(CMakePackageConfigHelpers)
